airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/BigQueryUploaderFactory.java

```diff
@@ -64,15 +64,14 @@ public static BigQueryDirectUploader getUploader(final UploaderConfig uploaderCo
         datasetLocation);
 
     return getBigQueryDirectUploader(
-        uploaderConfig.getConfig(),
+        uploaderConfig.getBigQueryClientChunkSize(),
         targetTable,
         uploaderConfig.getBigQuery(),
         datasetLocation,
         recordFormatter);
   }
 
-  private static BigQueryDirectUploader getBigQueryDirectUploader(
-                                                                  final JsonNode config,
+  private static BigQueryDirectUploader getBigQueryDirectUploader(final Integer bigQueryClientChunkSize,
```

Comment:
let's be explicit that chunk size is a nullable param (i.e. it's intentional that it's a boxed Integer)
```suggestion
  /**
   * @param bigQueryClientChunkSize The chunk size from the config, or null if not set
   */
  private static BigQueryDirectUploader getBigQueryDirectUploader(final Integer bigQueryClientChunkSize,
```

Reply:
obsolete up the stack PR